receiver/mongodbreceiver/README.md

```diff
@@ -25,7 +25,6 @@ This receiver supports MongoDB versions:
 
 Mongodb recommends to set up a least privilege user (LPU) with a [`clusterMonitor` role](https://www.mongodb.com/docs/v5.0/reference/built-in-roles/#mongodb-authrole-clusterMonitor) in order to collect metrics. Please refer to [lpu.sh](./testdata/integration/scripts/lpu.sh) for an example of how to configure these permissions.
 
-Collecting metrics `mongodb.global_lock.time` and `mongodb.index.access.count` are only available for mongodb 4.0+.
```

Comment:
I see that `mongodb.global_lock.time` was not moved to the list below. Is that because it actually is available regardless of version?

Reply:
Yes, `mongodb.global_lock.time` is available on all supported versions 2.6, 3.0+, 4.0+ and 5.0 as shown in the pr description